sklearn/ensemble/_hist_gradient_boosting/grower.py

```diff
@@ -618,9 +618,8 @@ def split_next(self):
                 if child.is_leaf:
                     del child.histograms
 
-        # Release memory used by histograms as they are no longer needed for
-        # internal nodes once children histograms have been computed.
-        del node.histograms
```

Comment:
This was included in https://github.com/scikit-learn/scikit-learn/commit/e325f16d6a3b6e3b7810038bdc579477ea354cb3 because of a memory issue. To be safe, can you rerun the benchmark in https://github.com/scikit-learn/scikit-learn/pull/18334#issue-692132105 to make sure there are no regressions?